app/services/peopleforce_exporter.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class PeopleforceExporter
+  attr_accessor :errors, :employees_table, :jobs_table, :dependents_table, :candidates_table, :dayoffs_table
+
+  def initialize()
+    @errors = []
+    @employees_table = []
+    @jobs_table = []
+    @dependents_table = []
+    @candidates_table = []
+    @dayoffs_table = []
+  end
+
+  def perform
+    Person.where(status: Person::CURRENT_EMPLOYEE_STATUSES).each do |person|
+      @employees_table.append(EMPLOYEES_MAPPING.map { |mapping| mapping[:value].call(person).to_s })
+      @jobs_table.append(JOBS_MAPPING.map { |mapping| mapping[:value].call(person).to_s })
+      @dependents_table.append(DEPENDENTS_MAPPING.map { |mapping| mapping[:value].call(person).to_s })
```

Comment:
we do not have this data in HRMS, i recommend removing this code

Reply:
done